### DISCUSSION TITLE
Shouldn't merging a PR flag me as a contributor?

### DISCUSSION THREAD

**Discussion by sasq64** [mruby/mruby#6034] [General]
Shouldn't this PR have added me to the list of contributors, or am I missing something?

https://github.com/mruby/mruby/pull/5596

**Comment by matz** [comment]
I checked the `AUTHORS` and found the following line regarding #5596:

```
   3 Jonas Minnberg (@sasq64)
```

Should we update the entry?

**Reply by sasq64** [reply]
I actually meant what github considers a contributor. I thought github would automatically add me when a PR by me was merged?

**Reply by matz** [reply]
I am not sure. I thought it should.

**Comment by matz** [comment]
After some googling, it was discovered that https://github.com/mruby/mruby/graphs/contributors displays only the top 100 contributors who have made at least 4 commits. As you have made only 3 commits, it is likely that making a few more commits will allow you to be listed among the top contributors.
